server/bleep/Cargo.toml

```diff
@@ -110,6 +110,9 @@ jsonwebtoken = { version = "8.3.0", features = ["use_pem"] }
 tiktoken-rs = "0.4.2"
 semver = { version = "1", features = ["serde"] }
 
+# vendored deps
+openssl-sys = { version = "0.9.88", features = ["vendored"] }
```

Comment:
We don't currently even transitively depend on `openssl`, so I don't see a reason why we would bring this dependency in.

I also think `vendored` openssl likely won't work for us, as it doesn't use the system cert store, so we'll need additional work to make HTTPS through it functional.

Reply:
> We don't currently even transitively depend on `openssl`, so I don't see a reason why we would bring this dependency in.

What do you mean?

```
openssl-sys v0.9.88
├── native-tls v0.2.11
│   ├── hyper-tls v0.5.0
│   │   └── reqwest v0.11.18
│   │       ├── bleep v0.4.5
...
```

> I also think `vendored` openssl likely won't work for us, as it doesn't use the system cert store, so we'll need additional work to make HTTPS through it functional.

Do you have an example of this failing? I've tested on linux and Windows, and it doesn't seem to have any issues. We can use `openssl-probe` if there are broken examples.

Reply:
Oh, in this case a better fix would be to just use `rustls` for `rustls-tls-native-roots` feature on `reqwest` and avoid openssl altogether.

Reply:
It seems like there are a few things going on here.
 1. OpenSSL is occasionally pulled in through `hyper-tls`, depending on host platform
 2. `bleep` pulled in a few changes recently that subtly add a dependency here, but these are easy to fix and switch to `rustls` based on `cargo tree` output.
 3. `tauri` currently only supports vendored and unvendored OpenSSL, and our app config uses the vendored version, so the library (I have to assume) will be correctly initialized by Tauri. However, their `dev` branch already supports rustls, so I have to assume sooner or later they'll release it.
 
 I think the meaningful way forward here is to make sure `bleep` builds without `hyper-tls`, and deal with Tauri when they release their new feature configuration.

Reply:
As discussed offline, we can tackle this in another PR, given it seems certificates load correctly on all platforms